### ISSUE TITLE
[HW1] 파일 경로 관련 질문입니다.

### ISSUE THREAD

**Issue by hiddenpunch** [swsnu/swppfall2020#35]
run.py skeleton에서 main 안에서는 BabynameParser와 save를 상대경로로 부르고 있습니다.
그래서 제가 home directory에서 terminal로 run.py를 실행을 하면 경로 오류가 나는데
검사하실때 terminal을 제 HW directory에서 실행하신다고 생각해도 괜찮을까요?

**Comment by hy00nc** [comment]
네 채점 시 HW directory에서 실행하는 것이 맞습니다.

**Comment by hiddenpunch** [comment]
감사합니다!
혹시 README 예시에서 BabynameParser 를 HW를 포함한 절대경로로 부르고 있는데, 이 역시 그냥 "babydata"로 부른다고 생각해도 될까요?

**Comment by hy00nc** [comment]
네 해당 예시는 directory를 인자로 받는다는 것을 보여드리기 위함이기 때문에 절대경로가 필수는 아닙니다.

**Comment by hiddenpunch** [comment]
아! 이해했습니다
답변 정말 감사합니다~!
